src/stringifiers/shared.js

```diff
@@ -151,10 +151,15 @@ FROM (
 
 export function orderingsToString(orderings, q, as) {
   const orderByClauses = []
-  for (const ordering of orderings) {
-    orderByClauses.push(
-      `${as ? q(as) + '.' : ''}${q(ordering.column)} ${ordering.direction}`
-    )
+  for (const { column, key, direction } of orderings) {
+    let sortKey
+    if(key) {
+      sortKey = key(as)
```

Comment:
Not exactly sure what else users of this might want, but, most of the other join monster functions that generate SQL take in a lot of arguments so the function implementor has lots of context to do crazy stuff. Should we pass more back to userland here?

Reply:
I can pass in all the context: table, column, and direction 👍 

I originally thought column would be unnecessary, because you have to set up the key function with a column, but perhaps someone would use a single function across all of their sort keys and would likely need all of that information.